std_msgs/Header header
int32[] ids
geometry_msgs/Pose[] locations
geometry_msgs/Pose[] predictions